.circleci/config.yml

```diff
@@ -4,10 +4,28 @@ orbs:
   terraform: circleci/terraform@2.1.0
 
 executors:
-  linux-machine:
+  cross-builder:
+    docker:
+      - image: quay.io/influxdb/cross-builder:go1.17.1-41d944d749b2c130119eff7efaaa9883ccffe5a7
+    resource_class: large
+  linux-amd64:
     machine:
       image: ubuntu-2004:202107-02
       resource_class: large
+  linux-arm64:
+    machine:
+      image: ubuntu-2004:202101-01
```

Comment:
Is there a reason to not use the newer `202107-02` image here?

Reply:
This is the newest available version for the ARM images (see [docs](https://circleci.com/docs/2.0/arm-resources/)). Not sure why they don't keep it 1:1 with the amd64 offerings...